### PR TITLE
Blogpost for PyDelhi's Meetup held on June 9, 2018.

### PR DESCRIPTION
**Pull request by im-gozmit**
Please review.

### CODE REVIEW

**Review by realslimshanky — COMMENTED** [review]
Please squash your commits. Rest LGTM.

File: content/2018-06-09-pydelhi-meetup.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
... Fueled, Noida for the PyDelhi Meetup. The meetup, as usual, started with the quick and interactive introductory session where each person presents gave his/her intro. [Vipul Gupta](https://twitter.com/vipulgupta2048), the host then gave a formal introduction about PyDelhi, talking about the various aspects of it. After which, we started with the talks. The first talk was given by Yug Khanna on the topic Introduction to Machine Learning & Deep Learning. He started the talk with the basics of the Python Libraries like Pandas and Numpy. Next, he proceeded to graph plotting libraries like Plotly and cufflinks and showed examples where he plotted graphs using these libraries in Jupyter Notebook. He explained how to get the HTML output of these graphs too. In the second part, he explained about Deep Learning where he explained Binary and Logistic Regression and their mathematical formulas using the famous Recognition of cat example. He ended the talk by telling about the uses of ML in daily life and answering numerous questions thrown by the gathering that was listening to the topic very intently. You can view the slides [here](https://prezi.com/p/wihqljm-gapk/ds-and-dl-9th-june-2018/).
```
Comment (dotgourav) [inline content/2018-06-09-pydelhi-meetup.md]: each person presents/gave(choose one)

File: content/2018-06-09-pydelhi-meetup.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+Then we had a snacks break for half an hour where people got the chance to interact with each other, network and discuss more with each other. [Shashank Kumar](https://twitter.com/realslimshanky) and [Vipul Gupta](https://twitter.com/vipulgupta2048) in the break took a small informal session about volunteering at Pydelhi and how we can contribute to it and also briefing about Google Summer of Code and how they contributed to the organization and their experience and journey till then. Later Yug who was also a Google Summer of Code student gave his views on the topic. Upcoming PyCon India 2018 was also briefly mentioned.
```
Comment (dotgourav) [inline content/2018-06-09-pydelhi-meetup.md]: Link PyCon India website here.

**Review by dotgourav — CHANGES_REQUESTED** [review]

**Comment by im-gozmit** [comment]
Thanks, @realslimshanky and @Gouravchawla for the edits.

**Comment by rajataaron** [comment]
@realslimshanky can you please check the PR once and merge it.

**Comment by realslimshanky** [comment]
@rajataaron I don't have merge access to this repo. @akshayaurora can you please help out here.

**Comment by akshayaurora** [comment]
PR is not in mergeable state, please fix conflicts.

File: content/2018-06-09-pydelhi-meetup.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+Slug: pydelhi-meetup-09-june-2018 
+Date: 2018-06-22 
+Category: Meetup 
+Author: Mayank  
+Email: mayankapex3@gmail.com
+Summary: ![Meetup at Fueled]({filename}/images/pydelhi-09-06-2018-1.jpg) PyDelhi members coming together and meeting up to learn, collaborate, contribute.
+
+![Yug's talk on Machine Learning]({filename}/images/pydelhi-09-06-2018-1.jpg)
+
+<<<<<<< HEAD
```
Comment (akshayaurora) [inline content/2018-06-09-pydelhi-meetup.md:11]: conflicts ?

**Comment by realslimshanky** [comment]
@im-gozmit please address reviews.

**Comment by aaqaishtyaq** [comment]
Hey @im-gozmit, can you address requested changes.

**Pull request merged**